tests/unit/test_var_info.py

```diff
@@ -862,3 +862,49 @@ def test_is_science_variable(self):
         self.assertFalse(dataset.is_science_variable(lat_variable))
         # Check that a science variable returns True
         self.assertTrue(dataset.is_science_variable(science_variable))
+
+    def test_get_missing_variable_attributes(self):
+        """Ensure that CF attributes for a variable is returned even if
+        the variable is not present in the source granule or dmrpp file
+        as long as there is a CF override for the variable defined in the
+        config json file
+
+        """
+        dataset = VarInfoFromDmr(
+            self.mock_dmr_two, 'FAKE99', config_file=self.config_file
+        )
+
+        with self.subTest('All CF attributes are retrieved for missing variable'):
+            self.assertDictEqual(
+                dataset.get_missing_variable_attributes('/group/variable'),
```

Comment:
I think it would be better to make the variable name something that really clearly doesn't exist. One of the other tests uses: `/non/existent/variable`.

Someone just looking at this test would have to dig through to find `tests/unit/data/mock_dmr_two.dmr` and then open it to check whether `/group/variable` existed.

Reply:
it still needs to be defined in the test_config.json
I replaced it with absent_override. Maybe that will indicate that it is a variable that is absent but has overrides in the json file.

Reply:
> it still needs to be defined in the test_config.json

Good point.

> Maybe that will indicate that it is a variable that is absent but has overrides in the json file.

It might be a little in the weeds. But, on the other hand, the word "absent" is there so that's the main thing.